### PR TITLE
Added documentation for A4A

### PR DESCRIPTION
**Pull request by bradfrizzell**
Added the Network Implementation Guide
Added the A4A Readme

### CODE REVIEW

**Comment by jasti** [comment]
LGTM. @lannka, can you please merge ASAP. Thanks.

**Comment by dknecht** [comment]
Can we add diagram that show inline add signing through a proxy?  Or do you prefer follow up pull?

**Comment by dknecht** [comment]
cc: @ampproject/cloudflare

**Comment by bradfrizzell** [comment]
Up to Vamsee, I know it's very important this be merged today.

**Comment by jasti** [comment]
Lets merge for now, but @dknecht feel free to send a follow up PR.

**Comment by bradfrizzell** [comment]
hongfei is on vacation as I understand, @erwinmombay  can you please merge ASAP? thanks :D

File: ads/google/a4a/docs/a4a-readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# A4A - AMP for Ads
```
Comment (cramforce) [inline ads/google/a4a/docs/a4a-readme.md]: I've seen marketing slides calling them `AMP Ads` now

**Pull request merged**